federated-learning/launch_local_servers.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/sh
+
+rm -f /tmp/config.json
+
+read -r -a instances <<< "$@"
+
+length=${#instances[@]}
+
+port=4000
+i=0
+echo "{" >> /tmp/config.json
+for instance in "${instances[@]}"
+do
+    echo "${instance}"
+    if [ "$i" = "$((length-1))" ]; then
+        echo "  \"$instance\": \"$port\"" >> /tmp/config.json
+    else
+        echo "  \"$instance\": \"$port\"," >> /tmp/config.json
+    fi
+    port=$((port+1))
+    i=$((i+1))
+done
+echo "}" >> /tmp/config.json
+
+pids=()
+for instance in "${instances[@]}"
+do
+    python -m tf_encrypted.player --config /tmp/config.json $instance > /dev/null 2> "${instance}-errors.log" &
+    pids+=( $! )
+done
+
+echo "Run: 'kill ${pids[@]}' to kill the server processes"
```

Comment:
specifying here that this should only be done on clean-up would be helpful -- can see some people confusing this as the next step they need to do instead of waiting until the tutorial is over. alternatively, writing a clean.sh one-liner that takes care of it for them would be even better